src/base64.c

```diff
@@ -7,7 +7,7 @@
  */
 
 #include <string.h>
-#include "orcania.h"
+#include <orcania.h>
```

Comment:
For the library source, I prefer keeping

```C
#include <string.h>
```

It's more relevant like this to me

Reply:
Indeed. It was a typo. `<some-api.h>` makes sense only in examples, e.g [mhd/doc/examples/hellobrowser.c](https://github.com/Karlson2k/libmicrohttpd/blob/3f5fe81d43728e9d7eecad00234ce244fa8211be/doc/examples/hellobrowser.c#L12). I've used MHD to show some practices, but I've seen them in other libraries too. I'm going to remove `<orcania.h>` from `src` and keep it only in the examples/tests.

Reply:
I made a typo on this one, I meant to keep the include like this in the library source:

```C
#include "orcania.h"
```
I don't know why it showed `string.h` instead, sorry

Reply:
Thanks for cleaning the changes. I'm going to compile and and check it again :sweat_smile:  ...

Reply:
Hey dude it was a typo. Fixed in https://github.com/silvioprog/orcania/commit/f4dfb2e179b7300240fb1cc0fd7c642514abaaf0 .